### ISSUE TITLE
FAQ section in LEARN?

### ISSUE THREAD

**Issue by w1g0** [ethereumclassic/ethereumclassic.github.io#878]
With the coming merge wpwrak formulated some recurring questions:

Q: My ETH miners just stopped. How can I switch them to ETC ?
Q: Which pool is best ?
Q: What wallet should I use for my mining returns ?

Also, for everyone (if ETHW actually launches):
Q: Where can I dump my ETHW while they're still worth something ?

**Comment by w1g0** [comment]
![2022-09-13_16h37_33](https://user-images.githubusercontent.com/67763069/189930328-fced496f-b5c2-4353-a874-45b6a962dc2f.png)

**Comment by IstoraMandiri** [comment]
@w1g0 

what would you suggest for "what wallet should I use for my mining returns"?

**Comment by w1g0** [comment]
> @w1g0
> 
> what would you suggest for "what wallet should I use for my mining returns"?

Not sure why this question was pointed towards me but I prefer either a Trezor or Ledger with a slight preference towards Ledger because of easy integration with 🕸️ 3.0 sites. But a Trezor is Open Source and leans more into the FOSS philosophy, it requires a bit more tweaking. In the end both of them are decent options. If you can't afford a hardware wallet look into Emerald, it's a clean and minimalist design. Whatever you choose don't send it to any custodial exchange wallet!

**Comment by IstoraMandiri** [comment]
Just to figure out a good general answer to this for the FAQs

**Comment by TheCrowbill** [comment]
I agree with @w1g0 on the Emerald wallet recommendation. It is an excellent wallet that just works.
